src/prisma/schema.prisma

```diff
@@ -2337,6 +2337,7 @@ model Project {
   primaryContractDeployTxHash      String?
   primaryContractChainId           Int?
   mintingWalletAddress             String?
+  ticketNo                         Int?                             @default(autoincrement())
```

Comment:
please spell out fields... and this is just for sunny, can we call this simply "sunnyAwardsNumber"

Reply:
done